### ISSUE TITLE
decrease a gauge

### ISSUE THREAD

**Issue by domenkozar** [sellerlabs/monad-metrics#15]
Hey @parsonsmatt thanks for this wonderful library!

I wonder is there any reason decrementing a gauge is not supported?

**Comment by parsonsmatt** [comment]
No reason at all - I'd be happy to accept a PR :)

**Comment by domenkozar** [comment]
```haskell
gaugeInc :: (MonadIO m, MonadMetrics m) => Text -> m ()
gaugeDec :: (MonadIO m, MonadMetrics m) => Text -> m ()
```

Does that make an acceptable API?

**Comment by domenkozar** [comment]
https://github.com/parsonsmatt/monad-metrics/pull/16

**Comment by parsonsmatt** [comment]
Released in `0.2.2.0`
